static/app/components/events/aiAutofix/index.spec.tsx

```diff
@@ -43,4 +48,32 @@ describe('AiAutofix', () => {
     expect(screen.getByText('Fixed the bug!')).toBeInTheDocument();
     expect(screen.getByText('View Pull Request')).toBeInTheDocument();
   });
+
+  it('renders the correct steps', () => {
+    render(
+      <AiAutofix
+        group={{
+          ...group,
+          metadata: {
+            autofix: {
+              status: 'COMPLETED',
+              completed_at: '',
+              created_at: '',
+              steps: [
+                {
+                  id: '1',
+                  title: 'I am processing',
+                  description: 'oh yes I am',
+                  status: 'PROCESSING',
+                },
+              ],
+            },
+          } as EventMetadataWithAutofix,
```

Comment:
Why do you need the cast?

Reply:
just realized the prop was typed as the wrong GroupWithAutofix